memgpt/memory.py

```diff
@@ -557,6 +566,7 @@ def _extract_date_from_timestamp(self, timestamp):
         match = re.match(r"(\d{4}-\d{2}-\d{2})", timestamp)
         return match.group(1) if match else None
 
+    # FIXME: Signature does not match the base class

```

Comment:
Not sure how important that is. But I think the IDE may have some idea why it made this a warning for me